packages/flutter_tools/lib/src/drive/web_driver_service.dart

```diff
@@ -247,102 +269,80 @@ Map<String, dynamic> getDesiredCapabilities(
   bool? headless, {
   List<String> webBrowserFlags = const <String>[],
   String? chromeBinary,
-}) {
-  switch (browser) {
-    case Browser.chrome:
-      return <String, dynamic>{
-        'acceptInsecureCerts': true,
-        'browserName': 'chrome',
-        'goog:loggingPrefs': <String, String>{
-          async_io.LogType.browser: 'INFO',
-          async_io.LogType.performance: 'ALL',
-        },
-        'goog:chromeOptions': <String, dynamic>{
-          if (chromeBinary != null)
-            'binary': chromeBinary,
-          'w3c': true,
-          'args': <String>[
-            '--bwsi',
-            '--disable-background-timer-throttling',
-            '--disable-default-apps',
-            '--disable-extensions',
-            '--disable-popup-blocking',
-            '--disable-translate',
-            '--no-default-browser-check',
-            '--no-sandbox',
-            '--no-first-run',
-            if (headless!) '--headless',
-            ...webBrowserFlags,
-          ],
-          'perfLoggingPrefs': <String, String>{
-            'traceCategories':
-            'devtools.timeline,'
-            'v8,blink.console,benchmark,blink,'
-            'blink.user_timing',
+}) =>
```

Comment:
I wouldn't use the arrow here, what comes after it is ~80 lines long

* https://dart.dev/tools/linter-rules#prefer_expression_function_bodies

Reply:
where?

Reply:
@kevmoo in the `getDesiredCapabilities` function body:

```dart
@visibleForTesting
Map<String, dynamic> getDesiredCapabilities(
  Browser browser,
  bool? headless, {
  List<String> webBrowserFlags = const <String>[],
  String? chromeBinary,
}) {
  return switch(...) {
  }
}